.ng-dev/pull-request.mts

```diff
@@ -3,9 +3,7 @@ import {PullRequestConfig} from '../ng-dev/pr/config/index.js';
 /** Configuration for interacting with pull requests in the repo. */
 export const pullRequest: PullRequestConfig = {
   githubApiMerge: false,
-  mergeReadyLabel: 'action: merge',
-  caretakerNoteLabel: 'merge note',
-  commitMessageFixupLabel: 'needs commit fixup',
+  requiredStatuses: [{name: 'test', type: 'check'}],
```

Comment:
Should this be `CI / test (pull_request)`?

Reply:
Verified with GitHub's API that it would just be `test`.

```graphql
query {
  repository(owner:"angular", name:"dev-infra"){
    pullRequest(number:1515){
      commits(last:1)	{
        nodes{
          commit {
            abbreviatedOid,
            statusCheckRollup{
              contexts(first: 100) {
                nodes {
                  ... on CheckRun {
                    name
                    
                  }
                  ... on StatusContext {
                    context
                  }
           
                }
              }
            }
          }
        }
      }
    }
  }
}
```

Reply:
weird, thats kind of annoying that its different than the rest of things.

Reply:
Haha yeah, there seems to be some special logic for GHA I think.. Wasn't able to find anything in the responses of `CheckRun`